qemu/tests/virtio_fs_share_data.py

```diff
@@ -455,7 +455,23 @@ def viofs_svc_stop_start(action, cmd, expect_status):
                     session.cmd(cmd_make % fs_dest, io_timeout)
                     status, output = session.cmd_status_output(
                         cmd_pjdfstest % fs_dest, io_timeout)
-                    if status != 0:
+                    failed_test = output.split("-------------------")[1]\
+                                        .split("Files=")[0]
+                    # ignore the specific failed cases from pjdfstest
+                    ignore_cases = params.objects("pjdfstest_blacklist")
+                    cases_in_output = list(re.findall(r"/mnt.*\.t", failed_test))
```

Comment:
1. What about use re.findall to find all test cases here, i.e. no need line458, e.g. re.findall(r'/mnt.+? Failed:.+?\)', output)
2. can we use params['fs_dest'] here instead of hard code '/mnt'
3. Please note that here we get all cases instead of the failed ones, refer to Xiaoling's paste below:
 /mnt/myfs/pjdfstest/tests/chown/00.t (Wstat: 0 Tests: 1323 Failed: 0)
4. loop all cases, re.search the failed one(remove /mnt/myfs/ here) when Failed number > 0,  then we get the failed case list
5.  use set(failed cases) - set(black list), then we can get the unexpected failed cases

Reply:
> 1. What about use re.findall to find all test cases here, i.e. no need line458, e.g. re.findall(r'/mnt.+? Failed:.+?)', output)
> 
>     2. can we use params['fs_dest'] here instead of hard code '/mnt'

Yes, the `cases_in_output = list(re.findall(r"/mnt.*\.t", failed_test))` maybe a bit of hard code, the mountpiont is changed in the future, then we can not cover it.

Reply:
hi @nickzhq , it seems you didn't notice yhong's comment here? 
We can click the 'Resolve conversation' button once we handle the comments, then it's quite clear to see if there is any comment to handle. Of course, if you would like to keep it without any code change, please also leave a msg to explain your thoughts, thanks a lot.

Reply:
Hello @zhencliu  , the issue yhong mentioned was solved( the code had been updated seven days ago, you can check the code at Files Changed ). The "Resolve" button was NOT clicked in time. Thanks for reminder.